install/kubernetes/helm/istio-remote/templates/endpoints.yaml

```diff
@@ -62,3 +62,41 @@ subsets:
     port: 9125
     protocol: UDP
 ---
+apiVersion: v1
+kind: Endpoints
+metadata:
+  name: istio-telemetry
+  namespace: istio-system
+subsets:
+- addresses:
+  - ip: {{ .Values.global.telemetryEndpoint }}
+  ports:
+  - name: prometheus
+    port: 42422
+    protocol: TCP
+  - name: http-monitoring
+    port: 9093
+    protocol: TCP
+  - name: http2-mixer
+    port: 9091
+    protocol: TCP
+  - name: tcp-mtls
+    port: 15004
+    protocol: TCP
+---
+apiVersion: v1
+kind: Endpoints
```

Comment:
what needs an ingress endpoint on the remotes?

Reply:
@sdake see ingress_test.go

@john-a-joyce I agree with you. I think that we can fix that in a separate PR.

Reply:
I am ok with fixing in a separate PR,  but we really shoudl do that not duplicate the ports in multiple places they have been pretty dynamic in a relative sense.